.github/workflows/tests.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install apt dependencies (Linux)
-      run: sudo apt-get install unrar
+      run: sudo apt-get install unrar libgl1-mesa-glx xvfb
```

Comment:
Let's remove this apt install and leverage PyVista's headless display action to set up everything appropriately on each OS